Crash when selecting One Dark theme with window popped out

### Is there an existing issue for this?

- [x] I have searched the [existing issues](https://github.com/4ian/GDevelop/issues)

### Describe the bug

Opening an event sheet or extension as a second window causes a crash when selecting the One Dark theme. It only happens with this theme when another window is popped out.

From my testing, this does not affect the web version of the app. 

I do not have a windows/mac device to test on.



### Steps to reproduce

Go to event sheet/extension tab
Right click tab -> Pop out into Separate Window
Change theme to One Dark

### GDevelop platform

Desktop

### GDevelop version

5.6.266

### Platform info

<details>

*OS (e.g. Windows, Linux, macOS, Android, iOS)*
>Ubuntu 26.10 appImage

*Browser(For Web) (e.g. Chrome, Firefox, Safari)*
>

*Device(For Mobile) (e.g. iPhone 12, Samsung Galaxy S21)*
>Asus VivoBook

</details>


### Additional context

<img width="682" height="526" alt="Image" src="https://github.com/user-attachments/assets/462c3677-cd62-4c49-9eb5-7d5de6ef0b8a" />

Console Output below:

 TypeError: Cannot read properties of undefined (reading 'replace')
    at d (ColorTransformer.js:88:34)
    at w.setWindowBackgroundColor (Window.js:145:40)
    at PoppedOutEditorContainerWindow.js:400:14
    at ol (react-dom.production.min.js:244:332)
    at Su (react-dom.production.min.js:286:111)
    at react-dom.production.min.js:283:409
    at Tu (react-dom.production.min.js:281:398)
    at uu (react-dom.production.min.js:273:439)
    at zo (react-dom.production.min.js:127:105)
    at du (react-dom.production.min.js:274:266)
(anonymous) @ rrweb-plugin-console-record.js:145
hs @ react-dom.production.min.js:189
i.componentDidCatch.r.callback @ react-dom.production.min.js:190
Ui @ react-dom.production.min.js:144
Cl @ react-dom.production.min.js:261
_l @ react-dom.production.min.js:260
bl @ react-dom.production.min.js:259
(anonymous) @ react-dom.production.min.js:283
Tu @ react-dom.production.min.js:281
uu @ react-dom.production.min.js:273
zo @ react-dom.production.min.js:127
du @ react-dom.production.min.js:274
Be @ react-dom.production.min.js:52
Wn @ react-dom.production.min.js:109
qt @ react-dom.production.min.js:74
Vt @ react-dom.production.min.js:73
handleMouseUp_ @ unknown
Show 16 more frames
Show less
ErrorBoundary.js:143 Error aac5cd61-ab42-4d75-a7c8-3207909aa93a caught by Boundary: TypeError: Cannot read properties of undefined (reading 'replace')
    at d (ColorTransformer.js:88:34)
    at w.setWindowBackgroundColor (Window.js:145:40)
    at PoppedOutEditorContainerWindow.js:400:14
    at ol (react-dom.production.min.js:244:332)
    at Su (react-dom.production.min.js:286:111)
    at react-dom.production.min.js:283:409
    at Tu (react-dom.production.min.js:281:398)
    at uu (react-dom.production.min.js:273:439)
    at zo (react-dom.production.min.js:127:105)
    at du (react-dom.production.min.js:274:266) 
    at yI (file:///tmp/.mount_GDevel71w1IG/resources/app.asar/www/static/js/2633.3377e92b.chunk.js:1:2085424)
    at C (file:///tmp/.mount_GDevel71w1IG/resources/app.asar/www/static/js/2633.3377e92b.chunk.js:1:3363446)
    at l (file:///tmp/.mount_GDevel71w1IG/resources/app.asar/www/static/js/5609.ed1c2bbb.chunk.js:2:259270)
    at p (file:///tmp/.mount_GDevel71w1IG/resources/app.asar/www/static/js/5609.ed1c2bbb.chunk.js:2:258661)
    at f (file:///tmp/.mount_GDevel71w1IG/resources/app.asar/www/static/js/2633.3377e92b.chunk.js:1:3591768)
    at m (file:///tmp/.mount_GDevel71w1IG/resources/app.asar/www/static/js/2633.3377e92b.chunk.js:1:3540284)
    at aI (file:///tmp/.mount_GDevel71w1IG/resources/app.asar/www/static/js/2633.3377e92b.chunk.js:1:2072825)
    at wI (file:///tmp/.mount_GDevel71w1IG/resources/app.asar/www/static/js/2633.3377e92b.chunk.js:1:2085628)
    at SI (file:///tmp/.mount_GDevel71w1IG/resources/app.asar/www/static/js/2633.3377e92b.chunk.js:1:2092823)
    at div
    at BW (file:///tmp/.mount_GDevel71w1IG/resources/app.asar/www/static/js/2633.3377e92b.chunk.js:1:2875984)
    at a (file:///tmp/.mount_GDevel71w1IG/resources/app.asar/www/static/js/2633.3377e92b.chunk.js:1:3281416)
    at x (file:///tmp/.mount_GDevel71w1IG/resources/app.asar/www/static/js/2633.3377e92b.chunk.js:1:2314)
    at m (file:///tmp/.mount_GDevel71w1IG/resources/app.asar/www/static/js/2633.3377e92b.chunk.js:1:3331267)
    at We (file:///tmp/.mount_GDevel71w1IG/resources/app.asar/www/static/js/2633.3377e92b.chunk.js:1:721923)
    at p (file:///tmp/.mount_GDevel71w1IG/resources/app.asar/www/static/js/2633.3377e92b.chunk.js:1:7647)
    at a (file:///tmp/.mount_GDevel71w1IG/resources/app.asar/www/static/js/2633.3377e92b.chunk.js:1:3356849)
    at C (file:///tmp/.mount_GDevel71w1IG/resources/app.asar/www/static/js/2633.3377e92b.chunk.js:1:172119)
    at g (file:///tmp/.mount_GDevel71w1IG/resources/app.asar/www/static/js/2633.3377e92b.chunk.js:1:58485)
    at p (file:///tmp/.mount_GDevel71w1IG/resources/app.asar/www/static/js/2633.3377e92b.chunk.js:1:144600)
    at l (file:///tmp/.mount_GDevel71w1IG/resources/app.asar/www/static/js/2633.3377e92b.chunk.js:1:2961183)
    at l (file:///tmp/.mount_GDevel71w1IG/resources/app.asar/www/static/js/2633.3377e92b.chunk.js:1:197866)
    at c (file:///tmp/.mount_GDevel71w1IG/resources/app.asar/www/static/js/2633.3377e92b.chunk.js:1:332832)
    at $ (file:///tmp/.mount_GDevel71w1IG/resources/app.asar/www/static/js/2633.3377e92b.chunk.js:1:133544)
    at f (file:///tmp/.mount_GDevel71w1IG/resources/app.asar/www/static/js/2633.3377e92b.chunk.js:1:78547)
    at v (file:///tmp/.mount_GDevel71w1IG/resources/app.asar/www/static/js/2633.3377e92b.chunk.js:1:192029)
    at h (file:///tmp/.mount_GDevel71w1IG/resources/app.asar/www/static/js/2633.3377e92b.chunk.js:1:135877)
    at p (file:///tmp/.mount_GDevel71w1IG/resources/app.asar/www/static/js/2633.3377e92b.chunk.js:1:211884)
    at C (file:///tmp/.mount_GDevel71w1IG/resources/app.asar/www/static/js/2633.3377e92b.chunk.js:1:15794)
    at u (file:///tmp/.mount_GDevel71w1IG/resources/app.asar/www/static/js/2633.3377e92b.chunk.js:1:27051)
    at l (file:///tmp/.mount_GDevel71w1IG/resources/app.asar/www/static/js/2633.3377e92b.chunk.js:1:303932)
    at Te (file:///tmp/.mount_GDevel71w1IG/resources/app.asar/www/static/js/2633.3377e92b.chunk.js:1:3252295)
    at Ie (file:///tmp/.mount_GDevel71w1IG/resources/app.asar/www/static/js/2633.3377e92b.chunk.js:1:719366)
    at t (file:///tmp/.mount_GDevel71w1IG/resources/app.asar/www/static/js/main.1da0b96c.js:2:324354)
    at S (file:///tmp/.mount_GDevel71w1IG/resources/app.asar/www/static/js/2633.3377e92b.chunk.js:1:3205967)
    at fe (file:///tmp/.mount_GDevel71w1IG/resources/app.asar/www/static/js/2633.3377e92b.chunk.js:1:686941)
    at C (file:///tmp/.mount_GDevel71w1IG/resources/app.asar/www/static/js/2633.3377e92b.chunk.js:1:3363446)
    at st (file:///tmp/.mount_GDevel71w1IG/resources/app.asar/www/static/js/2633.3377e92b.chunk.js:1:723776)
    at t (file:///tmp/.mount_GDevel71w1IG/resources/app.asar/www/static/js/5609.ed1c2bbb.chunk.js:2:799888)
    at S (file:///tmp/.mount_GDevel71w1IG/resources/app.asar/www/static/js/2633.3377e92b.chunk.js:1:3502240)
    at l (file:///tmp/.mount_GDevel71w1IG/resources/app.asar/www/static/js/5609.ed1c2bbb.chunk.js:2:259270)
    at p (file:///tmp/.mount_GDevel71w1IG/resources/app.asar/www/static/js/5609.ed1c2bbb.chunk.js:2:258661)
    at f (file:///tmp/.mount_GDevel71w1IG/resources/app.asar/www/static/js/2633.3377e92b.chunk.js:1:3591768)
    at t (file:///tmp/.mount_GDevel71w1IG/resources/app.asar/www/static/js/main.1da0b96c.js:2:323027)
    at Se (file:///tmp/.mount_GDevel71w1IG/resources/app.asar/www/static/js/2633.3377e92b.chunk.js:1:718255)
    at v (file:///tmp/.mount_GDevel71w1IG/resources/app.asar/www/static/js/main.1da0b96c.js:2:7830)
    at c (file:///tmp/.mount_GDevel71w1IG/resources/app.asar/www/static/js/2633.3377e92b.chunk.js:1:728780)
    at l (file:///tmp/.mount_GDevel71w1IG/resources/app.asar/www/static/js/2633.3377e92b.chunk.js:1:729466)
    at file:///tmp/.mount_GDevel71w1IG/resources/app.asar/www/static/js/5609.ed1c2bbb.chunk.js:2:750390
    at l (file:///tmp/.mount_GDevel71w1IG/resources/app.asar/www/static/js/2633.3377e92b.chunk.js:1:3488568)
    at mt (file:///tmp/.mount_GDevel71w1IG/resources/app.asar/www/static/js/2633.3377e92b.chunk.js:1:726401)
    at v (file:///tmp/.mount_GDevel71w1IG/resources/app.asar/www/static/js/main.1da0b96c.js:2:625979)
(anonymous) @ rrweb-plugin-console-record.js:145
(anonymous) @ ErrorBoundary.js:143
onError @ ErrorBoundary.js:276
value @ ErrorBoundary.js:57
i.componentDidCatch.r.callback @ react-dom.production.min.js:190
Ui @ react-dom.production.min.js:144
Cl @ react-dom.production.min.js:261
_l @ react-dom.production.min.js:260
bl @ react-dom.production.min.js:259
(anonymous) @ react-dom.production.min.js:283
Tu @ react-dom.production.min.js:281
uu @ react-dom.production.min.js:273
zo @ react-dom.production.min.js:127
du @ react-dom.production.min.js:274
Be @ react-dom.production.min.js:52
Wn @ react-dom.production.min.js:109
qt @ react-dom.production.min.js:74
Vt @ react-dom.production.min.js:73
handleMouseUp_ @ unknown
Show 16 more frames
Show less

--- a/newIDE/app/src/Utils/Window.js
+++ b/newIDE/app/src/Utils/Window.js
@@ -125,6 +125,13 @@ export default class Window {
   static setWindowBackgroundColor(newColor: string, targetDocument?: Document) {
     const doc = targetDocument || document;
 
+    if (!newColor) {
+      // Be defensive against themes missing a color (for instance, a
+      // newly added variable that an older or custom theme doesn't define):
+      // skip the update rather than crashing.
+      return;
+    }
+
     if (documentBackgroundColors.get(doc) === newColor) {
       // Avoid potentially expensive DOM query/modification if no changes needed.
       return;

